mk/spksrc.service.installer.dsm7

```diff
@@ -139,6 +140,24 @@ syno_user_add_to_legacy_group () {
     echo "${SYNOPKG_PKGNAME} has not been updated to DSM7 yet. syno_user_add_to_legacy_group() is no longer supported."
 }
 
+#
+# New DSM7 folders: copy no-verwrite + remove source
+# directories: @etc, @appdata, @apptemp and @apphome
+#
+syno_sync_app_folders () {
+    for appdir in $(find ${SYNOPKG_PKGDEST} -mindepth 1 -maxdepth 1 -not -empty -type d -name "@*");
+    do
+        case "${appdir##*@}" in
+           etc ) dest=/usr/syno/etc/packages/${SYNOPKG_PKGNAME};;
+             * ) dest=${SYNOPKG_PKGDEST}/../../@${appdir##*@}/${SYNOPKG_PKGNAME};;
+        esac
+
+        echo "$RSYNC --ignore-existing --remove-source-files ${SYNOPKG_PKGDEST}/@${appdir##*@}/ ${dest}" 1>&2
+        $RSYNC --ignore-existing --remove-source-files ${SYNOPKG_PKGDEST}/@${appdir##*@}/ ${dest}
+        $RM ${SYNOPKG_PKGDEST}/@${appdir##*@}
+    done
+}
+
```

Comment:
As @publicarray told we must avoid to use synology internal folder names (@etc, @appdata, @apptemp and @apphome)

Reply:
Done.
I've kept the `@` delimiter to differentiate between new vs pre-existing and keeps the code leaner.
```
/bin/rsync -avh --ignore-existing --remove-source-files /volume1/@appstore/tvheadend/@home/ /var/packages/tvheadend/home
/bin/rsync -avh --ignore-existing --remove-source-files /volume1/@appstore/tvheadend/@etc/ /usr/syno/etc/packages/tvheadend
/bin/rsync -avh --ignore-existing --remove-source-files /volume1/@appstore/tvheadend/@var/ /var/packages/tvheadend/var
/bin/rsync -avh --ignore-existing --remove-source-files /volume1/@appstore/tvheadend/@tmp/ /var/packages/tvheadend/tmp
```